com.github.yucefsourani.albasheer-electronic-quran-browser.json

```diff
@@ -0,0 +1,85 @@
+{
+    "id": "com.github.yucefsourani.albasheer-electronic-quran-browser",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "command": "albasheer-browser",
+    "rename-icon": "albasheer",
+    "copy-icon": true,
+    "rename-desktop-file": "albasheer.desktop",
+    "finish-args": [
+            "--share=ipc", "--socket=x11",
+            "--device=dri",
+            "--socket=wayland",
+            "--share=network",
+            "--socket=pulseaudio",
+            "--filesystem=home:ro"
+    ],
+    "modules": [
+	{
+	  "name": "intltool",
+	  "cleanup": [ "*" ],
+	  "sources": [
+	    {
+	      "type": "archive",
+	      "url": "https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz",
+	      "sha256": "67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd"
+	    }
+	  ]
+	},
+    {
+        "name": "amiri-quran-fonts",
+        "buildsystem":"simple",
+        "sources": [
+            {
+                "type": "archive",
+                "url": "https://github.com/alif-type/amiri/archive/054d41d/amiri-054d41dc54992ff082808f6cfd2a068b59d3211c.tar.gz",
+                "sha256": "ea6e035baad47dbfcc6a0e9ae1f49d7eab3d7e47938ff2d37c77b4b17437c10e"
+            },
+            {
+                "type": "file",
+                "path": "amiri-fontconfig.conf"
+            },
+            {
+                "type": "file",
+                "path": "amiri-quran-fontconfig.conf"
+            }
+        ],
+        "build-commands": [
+            "pwd",
+            "ls -lh",
+            "mkdir -p /app/share/fonts/",
+            "cp *.ttf /app/share/fonts/",
+            "mkdir -p /app/etc/fonts/conf.d/",
+            "cp amiri-fontconfig.conf /app/etc/fonts/conf.d/",
+            "cp amiri-quran-fontconfig.conf /app/etc/fonts/conf.d/",
+            "fc-cache -fs"
+        ]
+    },
+    {
+        "name": "albasheer",
+        "buildsystem": "simple",
+        "build-options" : {
+                "env": {
+                    "PYTHON": "/usr/bin/python3"
+                }
+        },
+        "sources": [
+            {
+                "type": "git",
+                "url": "https://github.com/yucefsourani/albasheer-electronic-quran-browser"
+        },
+        {
+                "type": "file",
+                "path": "com.github.yucefsourani.albasheer-electronic-quran-browser.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html